server/src/controllers/organization_constants.controller.ts

```diff
@@ -31,24 +33,43 @@ export class OrganizationConstantController {
 
   @UseGuards(JwtAuthGuard)
   @Get()
-  async get(@User() user) {
-    const result = await this.organizationConstantsService.allEnvironmentConstants(user.organizationId);
+  async get(@User() user, @Req() req) {
+    const ability = await this.organizationConstantsAbilityFactory.organizationConstantActions(user, null);
+    const { decryptValue } = req.query;
+    const decrypt =
+      decryptValue === 'true' &&
+      (ability.can('createOrganizationConstant', OrganizationConstant) ||
+        ability.can('deleteOrganizationConstant', OrganizationConstant));
+    const result = await this.organizationConstantsService.allEnvironmentConstants(user.organizationId, decrypt);
     return { constants: result };
   }
 
   @UseGuards(IsPublicGuard)
   @Get(':app_slug')
-  async getConstantsFromApp(@App() app) {
-    const result = await this.organizationConstantsService.allEnvironmentConstants(app.organizationId);
+  async getConstantsFromApp(@App() app, @User() user, @Req() req) {
+    const ability = await this.organizationConstantsAbilityFactory.organizationConstantActions(user, null);
```

Comment:
This route is used for getting data for public apps, do we need `decryptValue` here?